generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>Similarly, can we break down the section heading to a shorter one? Thanks!</p><p> </p><p>Also, for the GPU memory size requirement, can we have the estimation of peak usage of GPU mem for the default setting?</p><p> </p><p>In my local test, the max GPU mem for the default (256^3) is:</p><p> </p><p>2024/07/08 07:07:04.849, 33676 MiB</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Added, thanks